asdf/_jsonschema/COPYING

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2013 Julian Berman
```

Comment:
That is a great question. I never looked at the atropy top-level licenses (thanks for mentioning it!).

I stuck it in the source directory so that it would be packaged with the source (I'm not sure if this is necessary but it does seem like it should satisfy the requirement that the "copyright notice and this permission notice shall be included in all copies or substantial portions of the Software".

It doesn't appear that astropy packages the licenses they include in the top-level folder (looking at the `DataTables` [license](https://github.com/astropy/astropy/blob/main/licenses/DATATABLES_LICENSE.rst) it mentions copyright by `Allan Jardine` and I find no mention of that name in the installed package).

What do you think about mentioning (perhaps in the README) that the vendorized jsonschema is subject to copyright and use restrictions by the license found in the `_jsonschema_` submodule (or hopefully something more elegantly worded). I don't see a license notice at the moment so perhaps adding it to the end of the README and also mentioning the broader license in LICENSE?

Reply:
I added a top level licenses directory with soft links to the included licenses (good idea!) and a link in the readme.